cmd/pbstatuser/main.go

```diff
@@ -20,25 +30,162 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+const ChannelBuffer = 32
+
+type SourceInfo struct {
+	Authentication clients.Authentication `json:"-"`
+
+	SourceID string `json:"resource_id"`
+
+	Identity identity.XRHID `json:"-"`
+}
```

Comment:
I believe this type never gets serialized to JSON, can we drop the tags from here?